src/containers/scheduleWrapper/scheduleWrapper.tsx

```diff
@@ -1,69 +1,80 @@
-import { useEffect, useState } from 'react';
-import ScheduleRow from '../scheduleRow';
-import { GridWrapper } from './scheduleWrapper.style';
-import SliceOptionsContext from '../../common/context/sliceOptionsContext';
-import ScheduleDayToggler from '../scheduleDayToggler';
-import Schedule from '../schedule';
-import { generateScheduleMatrix } from '../../common/utils/generateScheduleMatrix';
-import { useWeekContext } from '../../common/context/weekContext';
-import { useLecturerContext } from '../../common/context/lecturerContext';
-import { useGroupContext } from '../../common/context/groupContext';
+import { useEffect, useState } from "react";
+import ScheduleRow from "../scheduleRow";
+import { GridWrapper } from "./scheduleWrapper.style";
+import SliceOptionsContext from "../../common/context/sliceOptionsContext";
+import ScheduleDayToggler from "../scheduleDayToggler";
+import Schedule from "../schedule";
+import { generateScheduleMatrix } from "../../common/utils/generateScheduleMatrix";
+import { useWeekContext } from "../../common/context/weekContext";
+import { useLecturerContext } from "../../common/context/lecturerContext";
+import { useGroupContext } from "../../common/context/groupContext";
 
-const ScheduleWrapper = ({getData, contextType}) => {
-  const [sliceParams, setSliceParams] = useState(null);
+const ScheduleWrapper = ({
+  getData,
+  contextType,
+}: {
+  getData: any;
+  contextType: any;
+}) => {
+  const [sliceParams, setSliceParams] = useState<{
+    start: number;
+    end: number;
+  } | null>(null);
   const [data, setData] = useState(null);
 
-  const {currentWeek} = useWeekContext();
-  const {lecturer} = useLecturerContext();
-  const {group} = useGroupContext();
+  const { currentWeek } = useWeekContext()!;
```

Comment:
Using "!" to **destruct** context value, which initial value is `null` — you're shooting in your own leg.